blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/horizontal-links/default.jsx

```diff
@@ -39,6 +38,23 @@ const HorizontalLinksBar = ({ hierarchy, navBarColor, showHorizontalSeperatorDot
 	// the interior links and spans need to contrast with nav color scheme
 	const readableContrastingColor = navBarColor === "light" ? "#000" : "#fff";
 
+	const LinkBuilder = ({ item }) => {
+		const linkUrl = item.node_type === "link" ? item.url : item._id;
+		const linkText = item.node_type === "link" ? item.display_name : item.name;
+		const externalUrl = /(http(s?)):\/\//i.test(linkUrl);
+		const formattedURL = formatURL(linkUrl);
+
+		return (
+			<Link href={formattedURL} openInNewTab={externalUrl}>
+				{linkText}
+			</Link>
+		);
+	};
+	/**
+	TODO:
```

Comment:
Is this TODO to be done with this PR? or a future task?

Reply:
Future work....just notes for me as I move through sub tasks.  Will be removed before final PR for the entire ticket is presented